docs/source/it/migration.md

```diff
@@ -1,320 +1,313 @@
-<!--- 
```

Comment:
IDK what happened with this diff: the only change is the removal of L169-174 in the old file